doc/pcre2test.1

```diff
@@ -654,6 +654,7 @@ heavily used in the test files.
       convert_glob_separator=c  set glob separator character
       convert_length            set convert buffer length
       debug                     same as info,fullbincode
+      expand                    enable repetition feature in subject
```

Comment:
"subject" is the data that gets matched against, while `expand` affects the "pattern" or "needle".

maybe a description in imperative form like the ones used for `convert*` that says: `activate preprocessing of the pattern for expansion`?

Reply:
Dear @carenas, thank you for the great comment! One thing seems confusing: you kindly explained that `expand`  affects only the pattern and not the subject, but the output you posted from `pcre2test` seems to show that expansion is done on *both* the pattern and also the subject. Is that incorrect? If so, why does `/aaa/` match `aaa` in `\[a]{3}`?

Once that is clarified, I'm sure we can find a simple but accurate way to express what `expand` does. Of course, readers will still need to look at the detailed description if they want to understand fully.

Reply:
> show that expansion is done on _both_ the pattern and also the subject.

correct, and the syntax is similar, but note that there is no need for a modifier for the subject to be expanded, unlike what happens with the pattern when the modifier is missing:

```
  re> /\[a]{3}/B
------------------------------------------------------------------
        Bra
        [a
        ]{3}
        Ket
        End
------------------------------------------------------------------
```